tests/validator/validator.cpp

```diff
@@ -2041,11 +2048,13 @@ TEST(Validator, unitStandardUnitsWhichAreNotBaseUnits)
 
     validator->validateModel(m);
 
-    EXPECT_EQ_ERRORS(expectedErrors, validator);
+    EXPECT_EQ_ISSUES(expectedIssues, validator);
 }
 
 TEST(Validator, unitMultiplierFactorDifference)
 {
+    const std::vector<std::string> expectedIssues = {
+        "Variable 'v1' in component 'c1' has units of 'litre' and an equivalent variable 'v2' in component 'c2' has compatible but not equivalent units of 'big_barrel'. The mismatch is: multiplication factor of 10^-3."};
```

Comment:
Add a comma at the end of the issue string, so that ClangFormat can format this better.

Reply:
Done.